src/Test3/QueueTests/TestsQueue.cs

```diff
@@ -0,0 +1,37 @@
+using System;
+using System.Collections.Generic;
+using NUnit.Framework;
+using Test3;
+
+namespace QueueTests
+{
+    public class Tests
+    {
+        BlockingQueue<string> testQueue = new BlockingQueue<string>( );
```

Comment:
```suggestion
        BlockingQueue<string> testQueue = new();
```
Не пишите тип дважды